src/seals/testing/envs.py

```diff
@@ -122,7 +122,17 @@ def has_same_observations(rollout_a: Rollout, rollout_b: Rollout) -> bool:
     """True if `rollout_a` and `rollout_b` have the same observations."""
     obs_list_a = [step[0] for step in rollout_a]
     obs_list_b = [step[0] for step in rollout_b]
```

Comment:
Should this be checking for equal lengths before it does the `zip`?

Reply:
Good catch, I'll add a `len` check.